finality-aleph/src/sync/substrate/chain_status.rs

```diff
@@ -49,6 +50,13 @@ impl Display for Error {
                     hash
                 )
             }
+            MissingIndexedBody(hash) => {
```

Comment:
You never construct this, or do you?

It would only make sense if you expected the option to always be `Some` for blocks in the DB, but I'm not sure that's correct.

Reply:
oh come on, I was just fixing this :P

Reply:
遅い!